barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -270,7 +270,11 @@ template <typename Flavor> class SumcheckProver {
         // The evaluations of Libra uninvariates at \f$ g_0(u_0), \ldots, g_{d-1} (u_{d-1}) \f$ are added to the
         // transcript.
         if constexpr (Flavor::HasZK) {
-            transcript->send_to_verifier("Libra:evaluations", zk_sumcheck_data.libra_evaluations);
+            for (size_t idx = 0; idx < multivariate_d; idx++) {
+                const FF& libra_evaluation = zk_sumcheck_data.libra_evaluations[idx];
+                std::string libra_evaluation_label = "Libra:evaluation_" + std::to_string(idx);
```

Comment:
fixed a discrepancy in the way the prover and the verifer send/receive these evals

Reply:
actually fun fact the labels don't matter for anything other than debugging